src/style.css

```diff
@@ -1 +1,81 @@
 /* add styles here */
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300px;
+  line-height: 21.86px;
+  text-align: left;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(/src/fonts/Avenir-Book.ttf) format(ttf);
+  src: url(/src/fonts/Avenir-Heavy.ttf) format(ttf);
+  src: url(/src/fonts/Avenir.ttc) format(ttc);
+  font-weight: normal;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+  font-family: Avenir, sans-serif;
+  padding: 0;
+  width: 100vw; /* Полная ширина окна */
```

Comment:
Setting `width` to `100vw` can cause horizontal scrolling due to the vertical scrollbar width. Consider using `width: 100%;` or not setting the width at all as `body` is block-level and will naturally fill the width of its parent.